packages/components/src/entities/SampleAliquotDetailHeader.tsx

```diff
@@ -10,20 +10,22 @@ import { SAMPLE_STATE_COLUMN_NAME } from '../internal/components/samples/constan
 import { SampleStatusTag } from '../internal/components/samples/SampleStatusTag';
 import { getSampleStatus } from '../internal/components/samples/utils';
 import { UserDetailsRenderer } from '../internal/renderers/UserDetailsRenderer';
+import { ExpirationDateColumnRenderer } from '../internal/renderers/ExpirationDateColumnRenderer';
 
 interface SampleAliquotDetailHeaderProps {
     aliquotHeaderDisplayColumns: List<QueryColumn>;
     row: Map<string, any>;
 }
 
 export class SampleAliquotDetailHeader extends PureComponent<SampleAliquotDetailHeaderProps> {
-    renderDetailRow(label: string, data: any, key: any, userLookup = false) {
+    renderDetailRow(label: string, data: any, key: any, userLookup = false, isExpDate?: boolean) {
```

Comment:
minor: match existing param naming and typing:
```suggestion
    renderDetailRow(label: string, data: any, key: any, userLookup = false, useExpDate = false) {
```